OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/internal/backend/impl/UserBackendService.kt

```diff
@@ -72,18 +72,20 @@ internal class UserBackendService(
             throw BackendException(response.statusCode, response.payload, response.retryAfterSeconds)
         }
 
-        val responseJSON = JSONObject(response.payload)
-        var rywToken: String? = null
-        if (responseJSON.has("ryw_token")) {
-            rywToken = responseJSON.getString("ryw_token")
-        }
+        fun JSONObject.safeString(key: String): String? = if (this.has(key)) this.getString(key) else null
```

Comment:
This is already defined in `JSONObjectExtension.kt`, you can just add `import com.onesignal.common.safeString` to the top of this file to get this.